### DISCUSSION TITLE
setup github action

### DISCUSSION THREAD

**Discussion by faultables** [evilfactorylabs/gatekeeper#1] [Ideas]
tiap minggu (klo ga lupa) gw jalanin "bash [update.sh](https://github.com/evilfactorylabs/gatekeeper/blob/main/update.sh)" di lokal buat ...update blocklist? lalu push segala perubahan tersebut langsung ke branch `main` like a man. baru inget ada @anakmagang dan since doi udah naik gaji harusnya toil kek ginian bisa di handle sama @anakmagang ygy

ini bisa dibikin github actions yg do the job weekly dan biarkan si @anakmagang yg commit — and i need a hand to do so. anyone?

**Comment by kevanantha** [comment]
> tiap minggu (klo ga lupa) gw jalanin "bash [update.sh](https://github.com/evilfactorylabs/gatekeeper/blob/main/update.sh)" di lokal buat ...update blocklist? lalu push segala perubahan tersebut langsung ke branch `main` like a man. baru inget ada @anakmagang dan since doi udah naik gaji harusnya toil kek ginian bisa di handle sama @anakmagang ygy
> 
> 
> 
> ini bisa dibikin github actions yg do the job weekly dan biarkan si @anakmagang yg commit — and i need a hand to do so. anyone?

🙋

**Reply by faultables** [reply]
gas kita ga pakai issue2an

**Reply by kevanantha** [reply]
gw perlu credential-nya @anakmagang kagak? @faultables

**Reply by faultables** [reply]
harusnya kaga

**Reply by kevanantha** [reply]
expect-nya @anakmagang yang commit kan? current progress -> https://github.com/kevanantha/gatekeeper/actions/runs/3867831960/jobs/6592874685 @faultables

**Reply by faultables** [reply]
markites

![CleanShot 2023-01-17 at 23 31 09](https://user-images.githubusercontent.com/6561394/212956704-2e2b2ed0-d5f8-481e-a81e-8c275d30cdb6.png)

**Reply by faultables** [reply]
ok mantab harusnya aman ni tinggal nunggu mingdep

**Reply by kevanantha** [reply]
jadinya gini wkwk
![image](https://user-images.githubusercontent.com/31406355/213076070-29c7f91a-67ed-4a0b-b292-93904ac033e8.png)
salah kayanya @faultables

**Reply by faultables** [reply]
markifix
